data/starts.txt

```diff
@@ -0,0 +1,202 @@
+		Hyperdrive
+		"Ion Cannon"
+		"Pebble Core"
+		"Pulse Rifle"
+		"Quantum Keystone" 2
+
+# the start itself cannot initiate events
+mission "Greenwater start"
+	landing
+	invisible
+	to offer
+		has "hai space start"
+	on offer
+		event "not from boston"
+		event "grew up on greenwater"
+		log "Finally scraped together enough money for a down payment on a starship. The interest on the mortgage is exorbitant."
+		log "Factions" "Republic" `Hundreds of years ago, the independent territories in different parts of human space agreed to join together into a single democratic government, with Earth as its capital. The rise of the Republic ushered in a long period of peace and prosperity in human history.`
+			`Representation in the Republic Parliament is based on population. That means that some individual "Paradise Worlds" have more representatives than entire regions of space like the Dirt Belt that are more sparsely settled.`
+		log "Factions" "Syndicate" `The Syndicate is a megacorporation, the largest employer in human space. People who cannot find steady work elsewhere flock to the Syndicate factory worlds, where they are almost guaranteed to find a job. Although the Syndicate is technically part of the Republic, Syndicate worlds are governed directly by the corporation.`
+			`The Syndicate is a central part of the Republic's economy, but they are also well known for selling shoddy products and for doing damage to the environment on the worlds they control. Their treatment of workers is questionable, and their opposition to organized labor is legendary. Some Syndicate factory towns have even been accused of human rights abuses.`
+		log "Factions" "Pirates" `In poorer and more remote star systems, where the Navy seldom patrols, pirate attacks on merchant ships are frequent. Pirates are also known to attack large, unguarded convoys of freighters even in more populated areas. Most pirate fleets come from lawless worlds on the outskirts of human space.`
+		log "Factions" "Hai" `The Hai are a species of giant, intelligent rodents, who live to the north of human space. They allow any humans who discover their territory to live alongside them, and to trade with them and purchase their technology.`
+
+
+
+event "not from boston"
+	planet "New Boston"
+		description `New Boston appears to be a completely unremarkable world... but as you fly through its atmosphere, you get a sense of deja vu, as if you had once been familiar with this planet before.`
+		description `	Most of the land here is wet and marshy, and the coastal regions are prone to flooding. Some food crops are grown here, but most of the farmers instead plant flax and jute, which grow well in wet soil and are used to make linen and burlap cloth. The textile mills are hot and poorly ventilated, but they offer higher and more reliable income than farming.`
+
+event "grew up on greenwater"
+	planet "Greenwater"
+		add description `This is also the place where you grew up.`
```

Comment:
I would beef this up a little. As of now this feels kind of tacked on. The description of New Boston in the vanilla start has a much more significant mention of it being your homeworld.

Reply:
I mean sure thing is I dont really know what to say

Reply:
Maybe have a little bit about the planet from a local perspective like in the style of New Boston; culture, weather... (Probably I could suggest further changes to both New Boston and Greenwater descriptions)

Reply:
the thing is the description of greenwater already has these things in it and it's pretty fleshed out

Reply:
Hmm...
> Greenwater is one of the few worlds in Hai space that is home to a sizable human population. The temperature on most of the planet's surface is too warm for the Hai, but tolerable for human beings. Close to the equator are scattered human settlements for farming or fishing, which trade with the Hai cities closer to the poles for technology and equipment.

If I gave it a shot:
> Greenwater is unique among Hai worlds, not only in that it is home to a sizable human population, but also because it happens to be the world where you grew up. Most of the humans here live on the islands and archipelagoes across the equator of Greenrock where the Hai seldom visit on account of their fur coats, but from what you hear from humans who have come from "beyond the wormhole," the climate of Greenwater is like a "Paradise Planet" even without terraforming.
> While farming and fishing make up most of the human-centered economy here, a few merchants trade with the Hai up in the poles where the spaceport is located.

Of course it could probably use with some improvements.

Reply:
I love it!
~~I'm prob not the best choice for improving it, so I'll just apply it~~